shogun-lib/src/main/java/de/terrestris/shogun/lib/model/Application.java

```diff
@@ -70,7 +70,41 @@ public class Application extends BaseEntity {
     @ToString.Exclude
     @Schema(
         description = "The configuration to be considered by the client/application which may include all specific " +
-            "configurations required by the project."
+            "configurations required by the project.",
+        example = "{\n" +
+            "\"mapView\": {\n" +
+              "\"zoom\": 2,\n" +
+              "\"center\": [10, 51], \n" +
+              "\"extent\": null,\n" +
+              "\"projection\": \"EPSG:3857\",\n" +
+              "\"resolutions\": [\n" +
+                "8920,\n" +
+                "4480,\n" +
+                "2240,\n" +
+                "1120,\n" +
+                "560,\n" +
+                "350,\n" +
+                "280,\n" +
+                "140,\n" +
+                "70,\n" +
+                "28,\n" +
+                "14,\n" +
+                "7,\n" +
+                "2.8,\n" +
+                "1.4,\n" +
+                "0.7,\n" +
+                "0.28,\n" +
+                "0.07\n" +
+              "]\n" +
+            "},\n" +
+            "\"description\": \"An awesome shogun app\",\n" +
+            "\"theme\": {\n" +
+              "\"primaryColor\": \"#02203d\",\n" +
+              "\"secondaryColor\": \"#73b3fb\",\n" +
+              "\"complementaryColor\": \"#ffffff\",\n" +
+              "\"logoPath\": \"https://terrestris.github.io/fossgis2019/shared/img/shogun-qgis-talk/logo-shogun.png\"\n" +
+            "}\n" +
+          "}\n"
```

Comment:
Not sure if this example is actually needed as it is synthesized automatically by all subfields:

![image](https://user-images.githubusercontent.com/1137620/210981604-c9ba4495-b07b-431b-bbe6-040017eddf33.png)

Some fields (e.g. `name`) are missing anyway.